tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/PartialViews.spec.ts

```diff
@@ -0,0 +1,235 @@
+﻿import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
```

Comment:
The AliasHelper is unused
```suggestion
﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
```